### PR TITLE
added country drop down box for event creation

### LINKED ISSUE #136: Show Country/City suggestions
VMS has forms that requires user to input city, country e.t.c. There are a lot of Django packages which are easy and support suggesting country names e.t.c. Use them and suggest Country/City as the user enters values in the fields.

### PR DESCRIPTION
**Pull request by varunnaganathan**
closes #136.

Used an existing application called django-countries to get all the ISO 3166-1 countries as choices.
modified the event creation form to provide a drop down menu for countries than th previously existing country text input.Also modified the edit page to display previously selected country and a drop down to modify previous choice of country

### CODE REVIEW

File: vms/event/forms.py [hunk 1]
```diff
@@ -16,7 +18,12 @@ class Meta:
             'state',
             'city',
             'address',
+<<<<<<< HEAD
```
Comment (willingc) [inline vms/event/forms.py]: Merge conflict. Please remove unneeded lines such as ``<<<<<<<<<< HEAD`` and others related to the merge conflict.

**Comment by willingc** [comment]
@varunnaganathan Thank you for your contribution. I'm sorry about the delay in reviewing this pull request.

Thanks for submitting these changes. Some of the files will require additional correction before the pull request will be ready for acceptance.

Please review and remove the merge conflicts that still exist in these files. I commented on one for reference. Also, I am not clear why there are files with many extra newlines placed in the file. Perhaps you could explain why you felt the newlines were necessary.

Thanks.

**Comment by varunnaganathan** [comment]
Yes mam,will definitely remove the extra newlines as well as those related
to the merge conflict.1

**Comment by willingc** [comment]
@varunnaganathan Thanks for being polite. Though you are being polite, it's generally considered more polite and professional when working in open source to address others by their GitHub user name. A few advantages: 1) you don't risk offending someone, 2) the person will get notified if you use @username.

For example,

> Yes @willingc, will definitely remove the extra newlines as well as those related to the merge conflict.

**Comment by varunnaganathan** [comment]
I'm so sorry about that @willingc , will definitely take care of that in the future.Am quite new to open source contribution.

**Comment by tapasweni-pathak** [comment]
@willingc @varunnaganathan :+1:

**Comment by willingc** [comment]
@varunnaganathan Welcome to open source :smile: We're glad to have you contributing.

**Comment by varunnaganathan** [comment]
@willingc  I removed the merge conflicts and the unnecessary newlines.Please Review.

**Comment by varunnaganathan** [comment]
@tapasweni-pathak  , could someone review this PR.Thanks! :)

File: vms/event/forms.py (hunk 1 above)
Comment (tapasweni-pathak) [inline vms/event/forms.py]: @varunnaganathan They still exists.

File: vms/event/forms.py [hunk 2]
```diff
@@ -16,7 +18,12 @@ class Meta:
...
             'venue'
+=======
+            'venue',
+            'country2',
+>>>>>>> 93e903e... added country suggesting dropbox in event form
             ]
```
Comment (tapasweni-pathak) [inline vms/event/forms.py]: Can you please check these and remove?

**Comment by tapasweni-pathak** [comment]
Please clean the PR and rebase and reword the commits. Your commits should explain what your changes are doing. They should be easily trackable. So if someone wants to understand the flow they easily can.

**Comment by tapasweni-pathak** [comment]
@varunnaganathan Should I close this?

**Comment by varunnaganathan** [comment]
@tapasweni-pathak  , I'll rebase and re commit in a while.

**Comment by tapasweni-pathak** [comment]
@varunnaganathan Cool.

**Comment by tapasweni-pathak** [comment]
@varunnaganathan Any updates on this? Let us know if you need any help. :)

**Comment by tapasweni-pathak** [comment]
Closing this. Please reopen if rebased or submit a fresh pull request.

**Pull request closed without merging**